src/algokit/cli/project/run.py

```diff
@@ -118,6 +122,15 @@ def base_command(
                 default=None,
                 help="Limit execution to specific project types if executing from workspace. (Optional)",
             )(command)
+            command = click.option(
```

Comment:
I'm wondering if we invert this to `Execute workspace commands sequentially`, that's the feature we're trying to unlock and it also means it's default false as concurrent is the default.

Reply:
good point, inverted